PracticeWPF/PracticeWPF/MainWindow.xaml

```diff
@@ -0,0 +1,80 @@
+﻿<Window x:Class="PracticeWPF.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:PracticeWPF"
+        xmlns:view="clr-namespace:PracticeWPF.View"
+        xmlns:model="clr-namespace:PracticeWPF.Model"
+        xmlns:viewModel="clr-namespace:PracticeWPF.ViewModel"
+        mc:Ignorable="d"
+        Title="MainWindow" Height="450" Width="800">
+    <Window.DataContext>
+        <viewModel:MainWindowViewModel/>
+    </Window.DataContext>
+    <Window.Resources>
+        <DataTemplate x:Key="ProductTemplate" 
+                      DataType="{x:Type model:Product}">
+            <Grid>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition/>
+                    <ColumnDefinition/>
+                </Grid.ColumnDefinitions>
+                <Grid.RowDefinitions>
+                    <RowDefinition/>
+                </Grid.RowDefinitions>
+                <Image Grid.Column="0" 
+                       Grid.Row="0" 
+                       Margin="5" 
+                       Source="{Binding Path=Image}" 
+                       Width="75" Height="75"/>
+                <StackPanel Grid.Column="1" 
+                            Grid.Row="0" 
+                            Margin="5">
+                    <Label Content="{Binding Path=Brend}"/>
+                    <Label Content="{Binding Path=Model}"/>
+                </StackPanel>
+            </Grid>
+        </DataTemplate>
+    </Window.Resources>
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition/>
+            <ColumnDefinition/>
+        </Grid.ColumnDefinitions>
+        <Grid.RowDefinitions>
+            <RowDefinition/>
+            <RowDefinition Height="0.5*"/>
+            <RowDefinition Height="0.2*"/>
+        </Grid.RowDefinitions>
+        <ListView Margin="5" 
+                  Grid.Column="0" 
+                  Grid.Row="0" 
+                  Grid.ColumnSpan="2" 
+                  ItemsSource="{Binding Path=Products}" 
+                  ItemTemplate="{StaticResource ProductTemplate}" 
```

Comment:
No need to use ItemTemplate explicitly since you've defined DataTemplate for Product only.